### ISSUE TITLE
Explore new losses

### ISSUE THREAD

**Issue by LoicGrobol** [hopsparser/npdependency#59]
So far our training loss is the sum of the NLL loss for POS tagging, head prediction and deprel prediction (for the gold head), but it could be interesting to explore alternatives, see for instance <https://arxiv.org/abs/1805.06334> or <https://arxiv.org/abs/1209.2784>.

**Comment by LoicGrobol** [comment]
This will become necessary as we explore new MTL settings (such as treebank prediction #57). There are actually two points

- How to combine multiple losses (so far we simply sum)
- How to compute batch-level loss: so far we sum as well but normalization with batch size probably makes more sense to help decouple the speed of the descent (normally controlled by LR) and batch size.

More on this soon 👀

**Comment by LoicGrobol** [comment]
The loss used by Candito ([2022](https://aclanthology.org/2022.findings-acl.190.pdf)) seems very interesting for this.

**Comment by LoicGrobol** [comment]
While there is still room for improvement, this should be considered fixed for now when #78 is merged.
